kvmagent/kvmagent/test/vmmeta_testsuite/local/test_file_prefix_rebase.py

```
@@ -0,0 +1,278 @@
+
+    def _test_get_rebase_lock_path_no_extension(self):
+        self.assertEqual(
+            _get_rebase_lock_path('/a/imagecache/img'),
+            '/a/imagecache/img.vmmeta-lck')
+
+    # -- rebase sub-tests --------------------------------------------------
+
+    def _test_empty_old_prefix_raises(self):
+        vol = os.path.join(self.tmpdir, 'vol.qcow2')
+        _qemu_img_create(vol, '64M')
+        with self.assertRaises(Exception):
```

Ruff (0.15.7)

[warning] Do not assert blind exception: `Exception`

(B017)